Update to starter for change to sqlite3 (#26)
appacademy-starters/python-project-starter: README.md

2. Install dependencies

      ```bash
      pipenv install -r requirements.txt
      ```

3. Create a **.env** file based on the example with proper settings for your
   development environment

> 4. Setup your Sqlite3 database and make sure it matches your **.env** file

Reviewer: Students don't need to do anything to setup the database before using it

```suggestion
4. Make sure the SQLite3 database connection URL is in the **.env** file
```

> 5. Get into your pipenv, migrate your database, seed your database, and run your flask app

Reviewer: ```suggestion
5. Get into your pipenv, migrate your database, seed your database, and run your Flask app
```

| `flask seed all`      | Just a helpful syntax to run queries against the db to seed data. See the **app/seeds** folder for reference and more details |
| `heroku login -i`      | Authenticate your heroku-cli using the command line. Drop the -i to authenticate via the browser |
| `heroku authorizations:create` | Once authenticated, use this to generate an Oauth token |
| `heroku run -a <app name>` | Run a command from within the deployed container on Heroku |

> ### Dev Containers (OPTIONAL for M1 Users)

Reviewer: I don't think this section is necessary since students don't use Postgres in development anymore.

> The following instructions detail an *optional* development setup for M1 Mac users, especially those having issues with the `psycopg` package.  ***While we are not using `psycopg` in development anymore, the instructions for setting up a Dev Container are still provided if you want to use them.***

Reviewer: ```suggestion
**These instructions should only be followed if you are using `psycopg` or PostgresQL in development. If you are are using SQLite3 for development, these instructions will not apply to you.**

The following instructions detail an *optional* development setup for M1 Mac users, especially those having issues with the `psycopg` package and PostgresQL in their development environment.
```

5. Click "Open in Container" when VS Code prompts to open container in the bottom right hand corner. 
6. **Be Patient!** The initial install will take a LONG time, it's building a container that has postgres preconfigured and even installing all your project dependencies. (For both flask and react!)

   **Note:** This will take much less time on future starts because everything will be cached.

7. Once everything is up, be sure to make a `.env` file based on `.env.example` in both the root directory and the *react-app* directory before running your app. You do not need a `DATABASE_URL` in the `.env` file if you are using this Docker setup for development - the URL is already set in the image (see `.devcontainer/Dockerfile` for the URL).  ***You will need to make changes in the `.devcontainer/Dockerfile` if you want to use it with SQLite3 instead of Postgres, the required changes are listed in the comments in that file***  

8. Get into your pipenv, migrate your database, seed your database, and run your flask app

   ```bash
   pipenv shell
   ```

   ```bash
   flask db upgrade
   ```

   ```bash
   flask seed all
   ```

   ```bash
   flask run
   ```

> 9. To run the React App in development, checkout the [README](./react-app/README.md) inside the `react-app` directory.

Reviewer 2: I think there are some changes that would break the dev container and require some debugging and not work out of the box. I can work on refactoring it but I think that makes more sense in a separate branch to be merged in. For now, let's move this from the bottom of the README to a new README file in the .devcontainer folder to hide it from students for now. I'll refactor to be more informational rather than just a setup guide.

Author: Moved it to its own separate readme in the devcontianer folder

   **Note:** This will take much less time on future starts because everything will be cached.

> 7. Once everything is up, be sure to make a `.env` file based on `.env.example` in both the root directory and the *react-app* directory before running your app. You do not need a `DATABASE_URL` in the `.env` file if you are using this Docker setup for development - the URL is already set in the image (see `.devcontainer/Dockerfile` for the URL).  ***You will need to make changes in the `.devcontainer/Dockerfile` if you want to use it with Sqlite3 instead of Postgres, the required changes are listed in the comments in that file***  

Reviewer: ```suggestion
7. Once everything is up, be sure to make a `.env` file based on `.env.example` in both the root directory and the *react-app* directory before running your app. You do not need a `DATABASE_URL` in the `.env` file if you are using this Docker setup for development - the URL is already set in the image (see `.devcontainer/Dockerfile` for the URL).  ***You will need to make changes in the `.devcontainer/Dockerfile` if you want to use it with SQLite3 instead of Postgres, the required changes are listed in the comments in that file***  
```
